src/alphafold_impact/pipelines/data_collection_oa/pipeline.py

```diff
@@ -0,0 +1,23 @@
+from kedro.pipeline import Pipeline, node, pipeline
+
+from .nodes import retrieve_oa_works_for_concepts_and_years, save_oa_works_to_s3
+
+
+def create_pipeline(**kwargs) -> Pipeline:
+    return pipeline(
+        [
+            node(
+                func=retrieve_oa_works_for_concepts_and_years,
+                inputs=[
+                    "params:concept_ids",
+                    "params:publication_years",
+                ],
+                outputs="oa_works_for_concepts_list",
+            ),
+            node(
+                func=save_oa_works_to_s3,
+                inputs="oa_works_for_concepts_list",
+                outputs="oa_works_for_concepts_json",
```

Comment:
For the above function to be redundant, the output object must match one of the keys in catalog.yml. This may also be why you can't load it back up, ie. there are discrepancies between how you name inputs and outputs, and what is in the catalog.

Reply:
The outputs and the data catalog should now match.